jobsAdmin/JobsAdmin.go

```diff
@@ -74,7 +74,7 @@ var JobsAdmin interface {
 	// JobMgr returns the specified JobID's JobMgr
 	JobMgr(jobID common.JobID) (ste.IJobMgr, bool)
 	JobMgrEnsureExists(jobID common.JobID, level common.LogLevel, commandString string) ste.IJobMgr
-	JobMgrCreateWithLogger(jobID common.JobID, level common.LogLevel, commandString string, logger common.ILoggerResetable) ste.IJobMgr
+	JobMgrCreateWithLogger(jobID common.JobID, level common.LogLevel, commandString string) ste.IJobMgr
```

Comment:
let's leave the logger param o/w the WithLogger sounds incorrect.
we can then pass nil for the logger to achieve what we want.
I'll come back to this comment later.

Reply:
removed the code